js/views/userPageVw.js

```diff
@@ -307,6 +307,11 @@ module.exports = baseVw.extend({
           self.model.set({ownPage: self.options.ownPage});
           self.render();
           !self.currentItemHash && self.loadingDeferred.resolve();
+          
+          // Handle was requested
+          if (profile.handle) {
+            window.obEventBus.trigger('saveVisitedHandle', profile);
```

Comment:
Same comment as above... can the event type be more descriptive of what is happening? Maybe 'handleObtained' ...?